Sources/SnowManLogicSpecs/ViewSpec.swift

```diff
@@ -0,0 +1,106 @@
+import SnowManLogic
+import Swiftest
+
+class ViewSpec: Swiftest.Spec {
+  	let spec = describe("View logic") {
+		var io: MockIO!
+        	var view: View!
+	        before() {
+			io = MockIO()
+       		 	view = View(io: io)
+        	}
+        
+       		describe("#receiveGuess") {
+        		it("should return the user input if it is a string of letters") {
+            		io.stubbedUserInput = ["hello"]
+                    expect(view.receiveGuess(io.myReadLine)).to.equal("hello")
+           		}
+
+           		it("should return the user input if it is a single letter") {
+                		io.stubbedUserInput = ["a"]
+                		expect(view.receiveGuess(io.myReadLine)).to.equal("a")
+            		}
+
+            		it("should return an error message if the user input is a number") {   
+                		io.stubbedUserInput = ["1", "fail"]
+                		expect(view.receiveGuess(io.myReadLine)).to.equal("fail")
+            		}
+
+                	it("should return an error message if the user input is a non-letter character") {
+                        	io.stubbedUserInput = ["?", "fail"]
+                        	expect(view.receiveGuess(io.myReadLine)).to.equal("fail")
+                    	}
+
+                    	it("should return an error message if the user input is a combination of valid and invalid inputs") {
+                        	io.stubbedUserInput = ["?a", "fail"]
+                        	expect(view.receiveGuess(io.myReadLine)).to.equal("fail")
+                    	}
+
+                    	it("should return an error message if the user hits return") {
+                        	io.stubbedUserInput = ["", "fail"]
+                        	expect(view.receiveGuess(io.myReadLine)).to.equal("fail")
+                    	}
+        	}
+
+        	describe("#assignBlanks") {
+            		var gameWord: String!
+            		var correctGuesses: [String]!
+            
+            		before() {
+                		gameWord = "hello"
+                		correctGuesses = []
+            		}
+            
+            		it("should return only blanks for unguessed word") {
+                		expect(view.assignBlanks(gameWord, correctGuesses: correctGuesses)).to.equal("__  __  __  __  __ ")
+            		}
+
+            		it("should return no blanks if the word has been guessed") {
+                		correctGuesses = ["h", "e", "l", "o"]
+                		expect(view.assignBlanks(gameWord, correctGuesses: correctGuesses)).to.equal("h e l l o")
+            		}
+
+            		it("should return blanks and letters for partially guessed word") {
+                		correctGuesses = ["h", "e"]
+                		expect(view.assignBlanks(gameWord, correctGuesses: correctGuesses)).to.equal("h e __  __  __ ")
+            		}
+
+        	}
+
+	        describe("#displayRemainingGuesses") {
+        		it("should return the number of guesses remaining") {
+                    		var remainingGuesses = 4
```

Comment:
This could be my ignorance of how swift works...but, what object is this variable being set on?